src/sentry/utils/email/address.py

```diff
@@ -28,18 +27,30 @@ def get_from_email_domain() -> str:
     return _from_email_domain_cache[1]
 
 
-def email_to_group_id(address: str) -> int:
+def email_to_group_id(address: str) -> tuple[int | None, int | None]:
```

Comment:
nit: this could use a rename if we're extracting org ID as well

Reply:
Yes, I'll rename when I add in outbox messages.